pkg/web/api/connector_v1.go

```diff
@@ -128,10 +128,12 @@ func (c *ConnectorAPIv1) CreateConnector(
 		req.PipelineId,
 		fromproto.ConnectorConfig(req.Config),
 	)
-
 	if err != nil {
 		return nil, status.ConnectorError(cerrors.Errorf("failed to create connector: %w", err))
 	}
+	if created.Config.Name == "" {
```

Comment:
Maybe we should add a limit for the names as well? How does 64 characters sound? I would then apply this to both a pipeline name and a connector name.

And since we're adding validations, it might be good to add some for the ID as well. I propose a limit of 64 characters and maybe even a restriction of allowed characters like `[A-Za-z0-9\-\_]`. Thoughts, @hariso @maha-hajja @jmar910 @ahamidi?

Reply:
sounds good to me

Reply:
+1 from me as well.